app/assets/stylesheets/helpers/_publisher-metadata-with-logo.scss

```diff
@@ -9,6 +9,7 @@
 
   .metadata-column {
     padding-left: 0;
+    padding-top: govuk-spacing(3);
```

Comment:
I initially wondered what this was and I can see now it's part of the design of articles that the metadata block is the same width as the main content. As someone who isn't a designer, I can see why this design has been put in place but I question how necessary it is. I personally think it looks squashed now but I'm not gonna let my **opinions** block this PR.

Reply:
![image](https://user-images.githubusercontent.com/71266765/109330421-8aba0e80-7853-11eb-9e8f-3c2f22e36bd8.png)

Oh - this does the top 👆🏼 
Is this what you mean?

Reply:
Sort of. If you look at the div where this class is being used, which wraps around the metadata component call, there's also a `govuk-grid` class that's setting the width of the metadata. If you remove it it looks like this:
![Screenshot 2021-02-26 at 16 59 27](https://user-images.githubusercontent.com/64783893/109330861-0f0c9180-7854-11eb-9e65-58261ec8ad28.png)

I don't think there's a problem with that design over the current, `two-thirds` one and it means lower html verbosity because we can remove this wrapping div entirely. This however is a design change and I don't think this PR shouldn't get merged just because of my opinions on said design.

Reply:
![image](https://user-images.githubusercontent.com/71266765/109331711-14b6a700-7855-11eb-8761-9aa2cf4e840c.png)

I see what you mean re the code, I think this is intentional as the [visuals restriction also appear here](https://www.gov.uk/guidance/uk-carbon-capture-and-storage-government-funding-and-support) which is on `collections`. Course, still worth a noting

Reply:
That's fair. This feels like a wider decision by govuk designers. Feel free to resolve 👍